### DISCUSSION TITLE
Is it possible to run Lighthouse CI with a Chrome-based desktop app?

### DISCUSSION THREAD

**Discussion by andriiprima** [GoogleChrome/lighthouse-ci#945] [Q&A]
I have a chrome-based desktop app and I want to configure Lighthouse CI for it. Is it possible? If it is, is there any documentation on it? So far, I only managed to create a lighthouse report through this desktop app's DevTools.

**Comment by paulirish** [comment]
Nope. You need a page thats available on http or https
